modules/rosa-getting-started-environment-setup.adoc

```diff
@@ -8,10 +8,10 @@
 
 Before you create a {product-title} (ROSA) cluster, you must set up your environment by completing the following tasks:
 
-* Enable ROSA in your AWS account.
+* Verify ROSA prerequisites against your AWS and Red Hat accounts.
 * Install and configure the required command line interface (CLI) tools.
 * Verify the configuration of the CLI tools.
-* Verify that the AWS Elastic Load Balancing (ELB) service role exists.
-* Verify that the required AWS resource quotas are available.
+//* Verify that the AWS Elastic Load Balancing (ELB) service role exists.
```

Comment:
Should we remove this sentence and the following modules rather than hide them?

Are we planning to read these in the near future?